packages/rancher-istio/charts/README.md

```diff
@@ -7,6 +7,12 @@ A Rancher created chart that packages the istioctl binary to install via a helm
 ## Chart Dependencies
 - rancher-kiali-server-crd chart
 
+# Uninstallation Requirements 
+To ensure rancher-istio uninstalls correctly, you must uninstall rancher-istio prior to uninstalling chart dependencies (see installation requirements for chart dependencies). This is because all definitions need to be available in order to properly build the rancher-istio objects for removal.
+
+Example uninstallation error you might encounter if you remove dependent CRD charts prior to removing rancher-istio:
```

Comment:
nit: If you remove dependent CRD charts prior to removing rancher-istio, you may encounter the following error: